### DISCUSSION TITLE
Registration for Moose Training session Jan 16-17 2024

### DISCUSSION THREAD

**Discussion by hardud** [idaholab/moose#26521] [Q&A General] [answered]
I want to register for the Moose training session which is being held on 16-17 Jan 2024 at MIT. But I do not find an option to register on the website. Who should be the point of contact regarding this? Also, does anyone know if the session is going to be recorded and will it be available to access later?

**Comment by GiudGiud** [comment]
Hello

I'm the point of contact. 
The session will not be recorded and is in-person only. 

There are two full length training recordings available online already. While we will try to cover content differently in this January training, the existing recordings are good for use

Guillaume

**Reply by hardud** [reply]
Is there a way I can still register for the offline session on 16-17 Jan at MIT? I understand it's too late to ask for this now, but it will be very important to me to attend the sessions. 
Thank you.

**Reply by GiudGiud** [reply]
Are you in Massachusetts?
I can ask to re-open registration, should work

**Reply by hardud** [reply]
I am in University of Miami, Florida. But my Professor told me to apply for the training and book the tickets to Massachusetts. If registered, I assure that I will manage to reach Cambridge before time. I request you to please ask to re-open the registration.
Thanks.

**Reply by GiudGiud** [reply]
I just emailed. I ll let you know soon

**Reply by GiudGiud** [reply, marked as answer]
Registration should be re-opened now until tomorrow

**Reply by hardud** [reply]
Thank you so much for helping out. I just applied for the training. Looking forward to it.
